snooty/rstspec.toml

```diff
@@ -624,6 +624,7 @@ example = """.. mongo-web-shell::
 
 [directive."mongodb:openapi"]
 help = """Include a reStructuredText file's contents."""
+options.swagger-parse = "flag"
```

Comment:
Given that we're not going to be dedicating much more time to supporting the Realm version of this directive, perhaps we should make the Swagger version the default and use a flag to indicate the limited rST version is in use. (This would require a small content change in 1 place for Realm). What do you think of that?

Reply:
Good point! I could go ahead and make the change! 

I originally went with a `snooty-parse` flag but decided against it thinking that (1) it would save Realm docs the trouble of making that change and (2) it would make sense that a directive would be parsed by the parser (snooty) being used by default. But I think it makes more sense to change the flag if Realm is really the only one expected to be using this version of the directive.